src/content/questionnaire/ListView.tsx

```diff
@@ -111,78 +108,117 @@ const StyledActionButton = styled(Button)(
-    value: (a) => a?.organization?.name,
-    field: "organization.name",
+    renderValue: (a) => <TruncatedText text={a?.organization?.name} />,
+    fieldKey: "organization.name",
   },
   {
     label: "Study",
-    value: (a) => a.studyAbbreviation || "NA",
+    renderValue: (a) => (
+      <TruncatedText text={a.studyAbbreviation || "NA"} disableInteractiveTooltip={false} />
+    ),
     field: "studyAbbreviation",
   },
   {
     label: "Program",
-    value: (a) => a.programName || "NA",
+    renderValue: (a) => (
+      <TruncatedText text={a.programName || "NA"} disableInteractiveTooltip={false} />
+    ),
     field: "programName",
   },
   {
     label: "Status",
-    value: (a) => a.status,
+    renderValue: (a) => a.status,
     field: "status",
+    sx: {
+      width: "104px",
+    },
   },
   {
     label: "Submitted Date",
-    value: (a) => (a.submittedDate ? FormatDate(a.submittedDate, "M/D/YYYY h:mm A") : ""),
+    renderValue: (a) =>
+      a.submittedDate ? (
+        <StyledDateTooltip title={FormatDate(a.submittedDate, "M/D/YYYY h:mm A")} placement="top">
+          <span>{FormatDate(a.submittedDate, "M/D/YYYY")}</span>
+        </StyledDateTooltip>
+      ) : (
+        ""
+      ),
     field: "submittedDate",
     default: true,
+    sx: {
+      width: "161px",
+    },
   },
   {
     label: "Last Updated Date",
-    value: (a) => (a.updatedAt ? FormatDate(a.updatedAt, "M/D/YYYY h:mm A") : ""),
+    renderValue: (a) =>
+      a.updatedAt ? (
+        <StyledDateTooltip title={FormatDate(a.updatedAt, "M/D/YYYY h:mm A")} placement="top">
+          <span>{FormatDate(a.updatedAt, "M/D/YYYY")}</span>
+        </StyledDateTooltip>
+      ) : (
+        ""
+      ),
     field: "updatedAt",
+    sx: {
+      width: "181px",
+    },
   },
   {
     label: "Action",
-    value: (a, user) => {
-      const role = user?.role;
+    renderValue: (a) => (
+      <QuestionnaireContext.Consumer>
+        {({ user }) => {
+          const role = user?.role;
 
-      if (
-        (role === "User" || role === "Submitter" || role === "Organization Owner") &&
-        a.applicant?.applicantID === user._id &&
-        ["New", "In Progress", "Inquired"].includes(a.status)
-      ) {
-        return (
-          <Link to={`/submission/${a?.["_id"]}`} state={{ from: "/submissions" }}>
-            <StyledActionButton bg="#99E3BB" text="#156071" border="#63BA90">
-              Resume
-            </StyledActionButton>
-          </Link>
-        );
-      }
-      if (role === "Federal Lead" && ["Submitted", "In Review"].includes(a.status)) {
-        return (
-          <Link to={`/submission/${a?.["_id"]}`} state={{ from: "/submissions" }}>
-            <StyledActionButton bg="#F1C6B3" text="#5F564D" border="#DB9C62">
-              Review
-            </StyledActionButton>
-          </Link>
-        );
-      }
+          if (
+            (role === "User" || role === "Submitter" || role === "Organization Owner") &&
```

Comment:
This is the same subset of roles that is covered by `CanCreateSubmissionRequest`. Would it make sense to also just use that variable here too, or should it be left as-is?